apps/wpcom-block-editor/src/wpcom/features/live-preview/upgrade-button.tsx

```diff
@@ -102,17 +102,21 @@ export const LivePreviewUpgradeButton: FC< {
 			} );
 		};
 
-		if ( canvasMode === 'view' ) {
-			overrideSaveButtonClick( SAVE_HUB_SAVE_BUTTON_SELECTOR );
-			overrideSaveButtonHover( SAVE_HUB_SAVE_BUTTON_SELECTOR );
-		} else if ( canvasMode === 'edit' ) {
-			overrideSaveButtonClick( HEADER_SAVE_BUTTON_SELECTOR );
-			overrideSaveButtonHover( HEADER_SAVE_BUTTON_SELECTOR );
-		}
+		// Delay it to ensure the element is visible.
+		const timeout = window.setTimeout( () => {
+			if ( canvasMode === 'view' ) {
```

Comment:
Hmm, maybe we don't need the delay in `view` mode? I don't think it's broken. The delay itself makes it a bit confusing 🥲

Reply:
I think it's more consistent for both view and edit mode and it's safer for any dom manipulation regardless of the order of loading.